src/SnehalW/Leapyear.java

```diff
@@ -0,0 +1,37 @@
+package SnehalW;
+
+import java.util.Scanner;
+//3. Write a Java program that takes a year from user and print whether that year is a leap year or not(Divisible by 400 and 4 and not by 100)
+
+public class Leapyear {
+	int year;
+	boolean flag = false;
+
+	void Checkyear() {
+		Scanner sc = new Scanner(System.in);
```

Comment:
Never take input from service method always give user permission to do operation accordingly.
like if user wanted check operation for 1996..